proto/src/state/write.rs

```diff
@@ -0,0 +1,29 @@
+use crate::{Message, Protobuf};
+
+use std::fmt::Debug;
+
+use penumbra_storage::StateWrite;
+
+pub trait StateWriteProto: StateWrite + Send + Sync {
```

Comment:
@hdevalence maybe we would rather want to be consistent with the relationship between `StateRead` and `StateWrite`

```rust
pub trait StateWriteProto: StateWrite + StateReadProto + Send + Sync
```

thoughts?

Reply:
I don't think we need to put this bound in the trait definition, since it's covered by the blanket impls?